metrics-dropwizard/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'scala'
+}
+
+repositories {
+    mavenCentral()
```

Comment:
Why not `jcenter()`? Should we add `mavenCentral()` for all subprojects?

Reply:
the jcenter is already defined in the base build.gradle for all buildprojects. I would stick with that or add maven central if necessary in there... As Karsten proposes :)